kubejs/data/ptd/advancements/cogs.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parent": "ptd:root",
+  "criteria": {
+        "0": {
+            "trigger": "minecraft:inventory_changed",
```

Comment:
As cogwheels are made in the personal assembler (stone cutters table), I think a better trigger would be to see if it is crafted. Than just picked up, as it could be obtained from the crashed ship

Reply:
Agreed, though I have no idea how to do so.

Reply:
cogwheels are not made in the personal assembler though

Reply:
cogwheels are not made in the personal assembler though